tests/unit/release/loadConfig/release.spec.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 Open Assessment Technologies SA;
+ */
+jest.mock('../../../../src/log.js', () => ({
+    error: jest.fn(() => ({
+        exit: jest.fn()
+    })),
+    exit: jest.fn(() => ({
+        exit: jest.fn()
+    })),
+    doing: jest.fn(),
+    info: jest.fn(),
+    done: jest.fn(),
+    warn: jest.fn()
+}));
+
+jest.mock('inquirer', () => ({
+    prompt: jest.fn(() => ({ }))
+}));
+
+jest.mock('../../../../src/config.js', () => {
+    const originalModule = jest.requireActual('../../../../src/config.js');
+    //Mock the default export
+    return {
+        __esModule: true,
+        ...originalModule,
+        default: jest.fn(() => ({
+            load:  jest.fn(() => {}),
+            write:  jest.fn()
+        }))
+    };
+});
+
+jest.mock('open');
+
+import log from '../../../../src/log.js';
+import inquirer from 'inquirer';
+import open from 'open';
+import releaseFactory from '../../../../src/release.js';
+
+beforeAll(() => {
+    jest.useFakeTimers();
+});
+beforeEach(() => {
+    jest.spyOn(process, 'stdin', 'get').mockReturnValue({ isTTY: true });
+});
+afterEach(() => {
+    jest.clearAllMocks();
+});
```

Comment:
**issue:** It took me some time to understand why I had failed tests while you and other reviewers didn't. Actually, the reason why is that I have the environment variables set. [This code](https://github.com/oat-sa/tao-extension-release/blob/fix/TR-3312/clean-vulnerabilities/src/release.js#L390-L392) is not stubbed, and therefore it never enters in the case where the token is empty.

**suggestion:**
```suggestion
const OLD_ENV = process.env;
beforeEach(() => {
    jest.resetModules();
    process.env = {};
    jest.spyOn(process, 'stdin', 'get').mockReturnValue({ isTTY: true });
});
afterEach(() => {
    jest.clearAllMocks();
    process.env = OLD_ENV;
});
```